webpack.config.js

```diff
@@ -0,0 +1,64 @@
+import path from 'path';
+import webpack from 'webpack';
+import UglifyJSPlugin from 'uglifyjs-webpack-plugin';
+
+import {settings} from './gulp/config';
+
+const dev = {
+	entry: {
+		polyfills: './src/app/polyfills.js',
+		vendor: './src/app/vendor.js',
+		app: './src/app/index.js'
+	},
+	output: {
+		path: path.join(__dirname, settings.destinations.dev.app),
+		filename: '[name].bundle.js'
+	},
+	plugins: [
+		new webpack.optimize.CommonsChunkPlugin({
+			name: ['app', 'vendor', 'polyfills']
+		})
```

Comment:
The [ProvidePlugin](https://webpack.js.org/plugins/provide-plugin/) would be nice to have and probably should also be configurable in the package.json?